spark/src/main/scala/org/apache/comet/serde/aggregates.scala

```diff
@@ -151,17 +151,6 @@ object CometCount extends CometAggregateExpressionSerde[Count] {
 
 object CometAverage extends CometAggregateExpressionSerde[Average] {
 
-  override def getSupportLevel(avg: Average): SupportLevel = {
```

Comment:
we need to reflect it in `compatibility.md`

Reply:
It would be good to show user that some part is supported, decimals are more rare comparing to ints

Reply:
Sure . Let me update the documentation to reflect the right support then